src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
Hmm... Which looks better?

    Copy magnet link
    Copy name
or

    Copy name
    Copy magnet link

I'll check and add the icon. Is that a valid identifier or just your guess?